app/src/main/java/org/fossasia/openevent/fragments/LocationsFragment.java

```diff
@@ -61,9 +59,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
             @Override
             public void onRefresh() {
                 DataDownload download = new DataDownload();
-                dbSingleton.clearDatabase(DbContract.Microlocation.TABLE_NAME);
```

Comment:
Take the case....network is not available and u deleted the table...now so nothing to show